### PR TITLE
Move suggest constraints to conflict section

### PR DESCRIPTION
**Pull request by greg0ire**
The suggest section is supposed to contain text explaining why pick a
suggested package.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -36,11 +36,17 @@
     "config": {
         "sort-packages": true
     },
+    "conflict": {
```
Comment (Soullivaneuh) [inline composer.json]: Usually, the `conflict` key is after `suggest`.

Comment (greg0ire) [inline composer.json]: Fixed

File: composer.json [hunk 2]
```diff
@@ -37,10 +37,15 @@
...
-        "stof/doctrine-extensions-bundle": "^1.1",
-        "sonata-project/doctrine-orm-admin-bundle":"^2.2",
-        "sonata-project/doctrine-phpcr-admin-bundle": "^1.0"
+        "knplabs/doctrine-behaviors": "if you want to pick this translation backend",
+        "stof/doctrine-extensions-bundle": "if you want to pick the gedmo translation backend",
+        "sonata-project/doctrine-orm-admin-bundle": "if you persist entities",
+        "sonata-project/doctrine-phpcr-admin-bundle": "if you persist documents"
+    },
+    "conflict": {
+        "knplabs/doctrine-behaviors": "<1.0",
```
Comment (core23) [inline composer.json]: Please add a upper limit to the conflict deps

**Comment by core23** [comment]
Refs https://github.com/sonata-project/dev-kit/issues/10

File: composer.json (hunk 2 above)
Comment (Soullivaneuh) [inline composer.json]: `<1.0 || >=2.0`

Comment (greg0ire) [inline composer.json]: Fixed

File: composer.json [hunk 3]
```diff
@@ -37,10 +37,15 @@
...
-        "stof/doctrine-extensions-bundle": "^1.1",
-        "sonata-project/doctrine-orm-admin-bundle":"^2.2",
-        "sonata-project/doctrine-phpcr-admin-bundle": "^1.0"
+        "knplabs/doctrine-behaviors": "if you want to pick this translation backend",
+        "stof/doctrine-extensions-bundle": "if you want to pick the gedmo translation backend",
+        "sonata-project/doctrine-orm-admin-bundle": "if you persist entities",
+        "sonata-project/doctrine-phpcr-admin-bundle": "if you persist documents"
+    },
+    "conflict": {
+        "knplabs/doctrine-behaviors": "<1.0 || >= 2.0",
```
Comment (core23) [inline composer.json]: Please remove the space after `>=`

Comment (greg0ire) [inline composer.json]: done

**Comment by Soullivaneuh** [comment]
RTM. :+1:

File: composer.json [hunk 4]
```diff
@@ -37,10 +37,15 @@
         "sort-packages": true
     },
     "suggest": {
-        "knplabs/doctrine-behaviors": "^1.0",
-        "stof/doctrine-extensions-bundle": "^1.1",
-        "sonata-project/doctrine-orm-admin-bundle":"^2.2",
-        "sonata-project/doctrine-phpcr-admin-bundle": "^1.0"
+        "knplabs/doctrine-behaviors": "if you want to pick this translation backend",
+        "stof/doctrine-extensions-bundle": "if you want to pick the gedmo translation backend",
```
Comment (dbu) [inline composer.json]: If you translate orm entities with the gedmo extensions.

File: composer.json [hunk 5]
```diff
@@ -37,10 +37,15 @@
         "sort-packages": true
     },
     "suggest": {
-        "knplabs/doctrine-behaviors": "^1.0",
-        "stof/doctrine-extensions-bundle": "^1.1",
-        "sonata-project/doctrine-orm-admin-bundle":"^2.2",
-        "sonata-project/doctrine-phpcr-admin-bundle": "^1.0"
+        "knplabs/doctrine-behaviors": "if you want to pick this translation backend",
```
Comment (dbu) [inline composer.json]: If you translate orm entities with the knplabs behaviours.

Comment (greg0ire) [inline composer.json]: @dbu : this is merged already

Comment (dbu) [inline composer.json]: https://github.com/sonata-project/SonataTranslationBundle/commit/a96cd92288dcd72b353dea1a9ae6a08cae501519

Comment (greg0ire) [inline composer.json]: Thanks for that @dbu

**Pull request merged**